openamp/Makefile

```diff
@@ -45,8 +45,6 @@ $(COBJS): %$(OBJEXT): %.c
 $(BIN): $(OBJS)
 	$(call ARCHIVE, $@, $(OBJS))
 
-dirlinks::
```

Comment:
@xiaoxiang781216 we need `context` here and we need to add the openamp directory in Directories.mk

Reply:
Done.